src/utility/symmetrization.jl

```diff
@@ -61,7 +61,7 @@ end
 
 # rotation invariance
 
-Base.rotr90(t::PEPSTensor) = permute(t, ((1,), (5, 2, 3, 4)))
+#Base.rotr90(t::PEPSTensor) = permute(t, ((1,), (5, 2, 3, 4)))
 Base.rot180(t::PEPSTensor) = permute(t, ((1,), (4, 5, 2, 3)))
```

Comment:
Can we move all of these definitions together? `src/states/abstractpeps.jl` also has some of these